src/GafferUI/AnimationGadget.cpp

```diff
@@ -1524,6 +1466,11 @@ bool AnimationGadget::dragEnd( GadgetPtr gadget, const DragDropEvent &event )
 	m_moveAxis = MoveAxis::Both;
 	Pointer::setCurrent( "" );
 
+	if( ! updateHighlightingAndPreview( event ) )
+	{
+		return false;
```

Comment:
Why the return here?  Do none of the other changes made by this function require a redraw?

Reply:
good point, have updated code.